modules/understanding-vulnerability-scores.adoc

```diff
@@ -6,26 +6,26 @@
 = Understanding vulnerability scores
```

Comment:
do we have anything about severity or just CVSS score? If not, we definitely should, as we believe severity is a MUCH better indicator of risk than CVSS score. Can you create a ticket to address this?

Reply:
Do we follow the Red Hat ratings? https://access.redhat.com/security/updates/classification?extIdCarryOver=true&sc_cid=701f2000001OH74AAG

Reply:
Added this to ticket https://issues.redhat.com/browse/ROX-23212